Integrations/IronDefense/IronDefense.py

```diff
@@ -0,0 +1,369 @@
+                                err_msg))
+        else:
+            self.logger.debug('Successfully submitted status for alert ({})'.format(alert_id))
+            return 'Submitted status to IronDefense!'
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def fetch_incidents_command():
+    incidents = IRON_DEFENSE.fetch_incidents()
+    demisto.incidents(incidents)
+
+
+def test_module_command():
+    results = IRON_DEFENSE.test_module()
+    demisto.results(results)
+
+
+def update_analyst_ratings_command():
+    alert_id = demisto.getArg('alert_id')
+    severity = demisto.getArg('severity')
+    expectation = demisto.getArg('expectation')
+    comments = demisto.getArg('comments')
+    share_irondome_arg = demisto.getArg('share_comment_with_irondome')
+    share_irondome = True if share_irondome_arg.lower() == 'true' else False
+    results = IRON_DEFENSE.update_analyst_ratings(alert_id, severity=severity, expectation=expectation, comments=comments,
+                                                  share_irondome=share_irondome)
+    demisto.results(results)
+
+
+def add_comment_to_alert_command():
+    alert_id = demisto.getArg('alert_id')
+    comment = demisto.getArg('comment')
+    share_irondome_arg = demisto.getArg('share_comment_with_irondome')
+    share_irondome = True if share_irondome_arg.lower() == 'true' else False
+    results = IRON_DEFENSE.add_comment_to_alert(alert_id, comment=comment, share_irondome=share_irondome)
+    demisto.results(results)
+
+
+def set_alert_status_command():
+    alert_id = demisto.getArg('alert_id')
+    status = demisto.getArg('status')
+    comments = demisto.getArg('comments')
+    share_irondome_arg = demisto.getArg('share_comment_with_irondome')
+    share_irondome = True if share_irondome_arg.lower() == 'true' else False
+    results = IRON_DEFENSE.set_alert_status(alert_id, status=status, comments=comments,
+                                            share_irondome=share_irondome)
+    demisto.results(results)
+
+
+COMMANDS = {
+    'test-module': test_module_command,
+    # For now, we will not support fetching of incidents
+    # 'fetch-incidents': fetch_incidents_command,
+    'irondefense-rate-alert': update_analyst_ratings_command,
+    'irondefense-comment-alert': add_comment_to_alert_command,
+    'irondefense-set-alert-status': set_alert_status_command
+}
+COOKIE_KEY = 'user_sid'
+LOG_PREFIX = 'IronDefense Integration: '
+
+'''EXECUTION'''
+
+if __name__ == '__builtin__':
+    try:
+        # Globals
+        PARAMS = demisto.params()
+        CREDENTIALS = PARAMS.get('credentials')
+        HOST = PARAMS.get('ironAPIHost', 'localhost')
+        PORT = PARAMS.get('ironAPIPort', 443)
+        REQUEST_TIMEOUT = float(PARAMS.get('requestTimeout', 60))
+        DATA_STORE = DemistoDataStore(demisto)
+        LOGGER = DemistoLogger(demisto, LOG_PREFIX)
+
+        # initialize the IronDefense object
+        IRON_DEFENSE = IronDefense(requests.session(), HOST, PORT, CREDENTIALS, LOGGER,
```

Comment:
![image](https://user-images.githubusercontent.com/30797606/68105852-76c4fd80-fee8-11e9-9eb9-7feef58601dc.png)

please switch to `requests.Session` as the current one is depracated
```suggestion
        IRON_DEFENSE = IronDefense(requests.Session(), HOST, PORT, CREDENTIALS, LOGGER,
```